snapshots/gapminder/2023-09-18/under_five_mortality.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Child Mortality Rate (Gapminder)
+    description: |-
+      Gapminder’s historic estimates compiled by Klara Johansson and Mattias Lindgren.
+    date_published: 2015-01-01
+    version_producer: v7
+    title_snapshot: Child Mortality Rate (Gapminder)
+
+    # Citation
+    producer: Gapminder
+    citation_full: Child Mortality Rate, v7, Gapminder (2015)
```

Comment:
You should include here more details about the authors, something like:
"Compiled and documented by Klara Johansson and Mattias Lindgren from many sources but mainly based on..."